Assets/Scripts/Keyframe.cs

```diff
@@ -82,6 +82,8 @@ public class Message
     public Highlight[] highlights;
 
     public List<float> teleportAvatarBasePosition;
+
+    public bool episodeChanged;
```

Comment:
As usual, I will nitpick the naming. "Episode" is a very RL-specific term. Can you think of something more generic we're trying to convey here? Perhaps "scene changed"?

Reply:
>I don't understand the transition at 0:15/0:16. It still looked jarring. 
I pressed the episode change key multiple times while the effect was ongoing.